src/scene/comparisons.ts

```diff
@@ -28,6 +28,8 @@ export const canChangeSharpness = (type: string) =>
 
 export const hasText = (type: string) => type === "text";
 
+export const hasDecorators = (type: string) => type === "arrow";
```

Comment:
nit: not sure if this shouldn't be called `canHaveDecorators` as `hasDecorates` will suggest it has non-null decorators.

Reply:
Makes sense to me 👍